docs/tendermint-core/README.md

```diff
@@ -7,4 +7,16 @@ parent:
 
 # Overview
 
-See the side-bar for details on the various features of Tendermint Core.
+This section dives into the internals of Tendermint the implementation.
```

Comment:
I always thought of this fix whenever I came to the tendermint doc. Nice work :+1: